channels/serializers.py

```diff
@@ -175,9 +172,9 @@ def get_author_name(self, instance):
     def get_profile_image(self, instance):
         """Find the profile image for the post author"""
         user = self._get_user(instance)
-        if user and user.profile.image_small:
-            return user.profile.image_small
-        return urljoin(settings.SITE_BASE_URL, default_profile_image)
+        if user and user.profile:
```

Comment:
Could you roll this logic into `image_uri` so we have one single place where the profile image is determined?

Reply:
✅